core/src/main/scala/cats/Applicative.scala

```diff
@@ -38,6 +38,14 @@ import simulacrum.typeclass
   def replicateA[A](n: Int, fa: F[A]): F[List[A]] =
     sequence(List.fill(n)(fa))
 
+  /** Returns the given argument if `cond` is `false`, otherwise, unit lifted into F. */
+  def unlessA[A](cond: Boolean)(f: => F[A]): F[Unit] =
```

Comment:
what's the story on call-by-name vs `Eval[F[A]]`. Seems like the later is more the cats style (maybe have both, strict and Eval)?